### ISSUE TITLE
Members table: Change tab order for expand button and make the additional permissions text accessible to screenreaders

### ISSUE THREAD

**Issue by zandercymatics** [cisagov/getgov#3365]
### Issue description

Currently, the members table has some accessibility issues regarding the "Expand" button - as well as for its content.
Of note, the tab order "jumps" back up to the manage and action buttons when it is opened, rather than the domain links.
Additionally, the "additional permissions" text is not presently accessible for screenreaders.

### Acceptance criteria

- [ ] The tabindex for the expand button is changed such that when expanded, it navigates through the "domains assigned" domain list, if it exists
- [ ] Make the "additional permissions" text block accessible to screenreaders
   - This can be done with aria-describedby, or potentially by using tabindex as before

### Additional context

These two items are the result of our design review for the member pages.
For more context, see these slack threads:
- [Tab order issue](https://cisa-corp.slack.com/archives/C087NN9UW5C/p1737135225323829)
- [Additional permissions issue](https://cisa-corp.slack.com/archives/C087NN9UW5C/p1737135619248379) 

### Prioritization Comments
**WCAG Conformance Level:** AA
**WCAG Standard that applies:** 2.4.3 Focus Order, 1.3.1 Info and Relationships
**Critical Failure if Not Met?:** Yes
**Prioritization:** High/Must Do
**Reason:** Improper focus order and inaccessible content create significant usability barriers for keyboard and screen reader users, making it difficult to navigate and understand key information. Ensuring logical focus order and screen reader accessibility is critical to compliance and user experience. This issue is not linked to a current remediation plan, however it will likely be called out in upcoming compliance testing.

### Links to other issues

- 🔄 #3280
